core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4647,6 +4648,37 @@ paths:
                 $ref: '#/components/schemas/ErrorReport'
         500:
           $ref: '#/components/responses/RawlsInternalError'
+  /api/googleProjects:
+    post:
+      tags:
+        - googleProjects
+      summary: register google project in rawls
+      description: Adds a link between a billing project and a google project
```

Comment:
Would additionally noting that this operation assigns billing to the google project be appropriate here?